iam_units/data/emissions/SARGWP100.txt

```diff
@@ -44,18 +43,6 @@
     a_HFC32 = 650.0
     a_HFC41 = 150.0
     a_HFC4310mee = 1300.0
-    a_HFE125 = 14900.0
```

Comment:
Just to confirm & make explicit the mistake mentioned in the PR description & linked issue:

The source currently cited in our README (https://www.ghgprotocol.org/sites/default/files/ghgp/Global-Warming-Potential-Values%20%28Feb%2016%202016%29_1.pdf) includes this species (and those below) for AR4 and AR5, but not for SAR (the cell is blank).

The file metric_conversions.csv (initially reused from openscm-units, and now deleted in this PR), included the line:
```
Species,Symbol,SARGWP100,AR4GWP100,AR5GWP100
…
,HFE125,14900,12400,
```

So that was actually an erroneous transcription of the table, shifted left by one cell. The correct line should have been:
```
,HFE125,,14900,12400
 ```

In short, the removed lines in this generated file shouldn't have been there in the first place.